### DISCUSSION TITLE
Merging data of a two independent clusters

### DISCUSSION THREAD

**Discussion by cigolpl** [seaweedfs/seaweedfs#4131] [General]
I'd like to merge a data of two independent SeaWeedFS clusters to have only one cluster. My current idea to do it is to use Rclone which will copy files (file by file) directly from one cluster to the other. One cluster is using version 2.x and second is using newer 3.x.

Do you know a better, faster, simpler way to make such a merge ?

**Comment by chrislusf** [comment]
This is a good feature to have. Basically we need to change a set of volumes to new ids, and change metadata of affected files from old volume ids to new ones.

After that, merging can just copy volume files and merge filer metadata.
